angular-frontend/config/clientregistration.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   authServerSelector:
     matchLabels:
-      name: <your-authserver-uri>
+      name: <your-authserver-match-label>
```

Comment:
**[issue]**: this is still label `name`, where the entire key and value should be dynamic, e.g. `env=dev` or `app=sso`, etc.